### PR TITLE
Add alias to gaussian_filter

### PR DESCRIPTION
**Pull request by K-Monty**
Closes https://github.com/dask/dask-image/issues/192

Before you submit a pull request, check that it meets these guidelines:

1. The pull request should include tests.
2. If the pull request adds functionality, the docs should be updated. Put
   your new functionality into a function with a docstring, and add the
   feature to the list in README.rst.
3. The pull request should work for Python 3.6, 3.7, and 3.8. Check
   https://travis-ci.org/dask/dask-image/pull_requests
   and make sure that the tests pass for all supported Python versions.

### CODE REVIEW

**Comment by GenevieveBuckley** [comment]
Here are some comments, although you might like to wait a bit for responses to [this comment](https://github.com/dask/dask-image/issues/192#issuecomment-785531439) before trying it out.

1. Here's an example where we alias a function for deprecation reasons.

`sum` is an alias of `sum_labels`

https://github.com/dask/dask-image/blob/91fe6e1cb262058296c182ae9f9bd9b255aaebd9/dask_image/ndmeasure/__init__.py#L716-L719

Obviously, there's no need for a warning in this PR. Maybe the docstring could be as simple as `"""Alias of 'gaussian_filter'."""`

2. The second step is to add the alias function name to `__all__`

**Comment by K-Monty** [comment]
@GenevieveBuckley Thanks a lot for your comments! It seems like there is no objection(s) in [the comment](https://github.com/dask/dask-image/issues/192#issuecomment-785531439), so I'd given it another try. 
By the way, do I also need to pay attention to the checks? For my last pull request, it said "All checks have failed".

**Review by GenevieveBuckley — CHANGES_REQUESTED** [review]
Thanks @K-Monty, I've added some comments for you.

The checks all seem to have passed with your most recent commit, so I wouldn't worry about why they failed earlier unless we start seeing that again.

File: dask_image/ndfilters/_gaussian.py [hunk 1]
```diff
@@ -83,6 +84,21 @@ def gaussian_filter(image,
     return result
 
 
+def gaussian(image,
+             sigma,
+             order=0,
+             mode='reflect',
+             cval=0.0,
+             truncate=4.0):
+    """Alias of 'gaussian_filter'."""
```
[the remark this answers is not in the document]
Comment (K-Monty) [reply dask_image/ndfilters/_gaussian.py]: @GenevieveBuckley Thanks a lot for your helpful comments! I have made changes to the function as required :)

**Review by GenevieveBuckley — APPROVED** [review]

**Comment by GenevieveBuckley** [comment]
Ok, this looks good to me now!

I'm going to leave this PR open for another week, just to give @jakirkham & others a bit more time for [this discussion around design decisions](https://github.com/dask/dask-image/issues/192#issuecomment-785550085). If there are no strong objections in a week, then we'll merge this here (feel free to ping me if I forget).

**Pull request merged**